pom.xml

```diff
@@ -20,7 +20,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <java.version>1.7</java.version>
+    <java.version>1.8</java.version>
```

Comment:
Is the upgrade to Java 8 intended?

Reply:
Yes. Collectors are a Java 8 API.

Reply:
I imagine this is a safe change - with JDK 9 out and 10 on the way, most people should be using `javac8` at the very least by this point, right?

Reply:
iirc it was left on 1.7 for android compat reason ... some year(s) ago.

Reply:
Android tooling has required Java 8 since AGP 2.2 or 2.3. The biggest holdout for me was the IntelliJ platform but it's on Java 8 as of 2017.1.

Reply:
Perfect. This enables the migration to the JUnit Plaform! ;)